src/toast/Toast.stories.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {storiesOf} from "@storybook/react";
+import {State, Store} from "@sambego/storybook-state";
+import {Toast} from "./Toast";
+import {Button} from "../forms/button";
+import {Direction, MessageType} from "./Toast";
```

Comment:
You can import Toast here no need of two imports from /Toast

Reply:
Addressed comment